lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -1564,7 +1555,8 @@ def select(sql, name) # :nodoc:
           end
 
           ## Adjust Ruby string encoding, if requested
-          if rRows && enc=@connection_options[:ruby_string_encoding]
+          if enc=@connection_options[:ruby_string_encoding]
```

Comment:
good call, this was copied from the local changes that had been made in the localytics-rails project.